tests/licensedcode/test_match.py

```diff
@@ -873,10 +1026,10 @@ class TestCollectLicenseMatchTexts(FileBasedTesting):
 
     def test_get_full_matched_text_base(self):
         rule_text = u'''
-            Copyright {{some copyright}}
-            THIS IS FROM {{THE CODEHAUS}} AND CONTRIBUTORS
-            IN NO EVENT SHALL {{THE CODEHAUS}} OR ITS CONTRIBUTORS BE LIABLE
-            EVEN IF ADVISED OF THE {{POSSIBILITY OF SUCH}} DAMAGE
+            Copyright [[some copyright]]
```

Comment:
Simply replaced them as a workaround to avoid conflict with the key phrase syntax, does the `{{` and `}}` have another special meaning here?

Reply:
> do the {{ and }} characters have another special meaning here?

In a primitive iteration of license detection there was a notion of "template" where `{{10 some text}}` would mean that up to 10 words could be missed and the match still be valid.
- https://github.com/nexB/scancode-toolkit/blob/v1.0.0/src/licensedcode/models.py#L256
- https://github.com/nexB/scancode-toolkit/blob/v1.0.0/src/textcode/analysis.py#L146
- https://github.com/nexB/scancode-toolkit/blob/v1.0.0/src/textcode/analysis.py#L193

This was byzantine and did not work as expected at all!

Reply:
But this feature is not in anymore, is this test still relevant/correct? Since I changed the characters to `[[` and `]]` (which means nothing), and it didn't break :thinking:

Reply:
> But this feature is not in anymore, is this test still relevant/correct? Since I changed the characters to [[ and ]] (which means nothing), and it didn't break thinking Same for the other similar ones.

Yes it could be irrelevant but no harm done either to keep it: this is minor so your changes are fine.